src/Compilers/CSharp/Test/Syntax/Parsing/MemberDeclarationParsingTests.cs

```diff
@@ -742,5 +743,88 @@ public void GenericPublicTask_03()
                 EOF();
             }
         }
+
+        [Fact]
+        [CompilerTrait(CompilerFeature.InitOnlySetters)]
+        public void InitAccessor()
+        {
+            foreach (var options in new[] { TestOptions.Script, TestOptions.Regular })
+            {
+                UsingDeclaration("string Property { get; init; }", options: options);
+                N(SyntaxKind.PropertyDeclaration);
+                {
+                    N(SyntaxKind.PredefinedType);
+                    {
+                        N(SyntaxKind.StringKeyword);
+                    }
+                    N(SyntaxKind.IdentifierToken, "Property");
+                    N(SyntaxKind.AccessorList);
+                    {
+                        N(SyntaxKind.OpenBraceToken);
+                        N(SyntaxKind.GetAccessorDeclaration);
+                        {
+                            N(SyntaxKind.GetKeyword);
+                            N(SyntaxKind.SemicolonToken);
+                        }
+                        N(SyntaxKind.InitAccessorDeclaration);
+                        {
+                            N(SyntaxKind.InitKeyword);
+                            N(SyntaxKind.SemicolonToken);
+                        }
+                        N(SyntaxKind.CloseBraceToken);
+                    }
+                }
+                EOF();
+            }
+        }
+
+        [Fact]
+        [CompilerTrait(CompilerFeature.InitOnlySetters)]
+        public void InitAndSetAccessor()
+        {
+            foreach (var options in new[] { TestOptions.Script, TestOptions.Regular })
+            {
+                UsingDeclaration("string Property { init set; }", options: options,
```

Comment:
>string Property { init set; } [](start = 34, length = 29)

And what shape are we going to have for scenarios "string Property { init; set; }" and "string Property { set; init; }"?